api/src/wfl/executor.clj

```diff
@@ -160,17 +146,17 @@
   [{:keys [skipValidation
            workspace
            methodConfiguration
-           methodConfigurationVersion
            fromSource] :as executor}]
-  (when-not skipValidation
-    (firecloud/workspace-or-throw workspace)
-    (when-not (= "importSnapshot" fromSource)
-      (throw
-       (UserException. "Unsupported coercion" (util/make-map fromSource))))
-    (let [m (firecloud/method-configuration workspace methodConfiguration)]
-      (error-on-method-config-version-mismatch m methodConfigurationVersion true)
-      (throw-unless-dockstore-method (:methodRepoMethod m))))
-  executor)
+  (if skipValidation
```

Comment:
Lift the `skipValidation` check into the `defmethod`?
Mark this `^:private`?
Rename `request` parameter to `executor`?  (I did not tug on this.)